component/src/main/java/me/androidbox/component/agenda/AddVisitorDialog.kt

```diff
@@ -73,11 +76,18 @@ fun AddVisitorDialog(
                 }
             )
 
-            if(!isEmailVerified) {
+            if(!isEmailVerified or isAlreadyAdded) {
```

Comment:
In general, I'd recommend to use the real logical operators ( `||`) for this. For booleans it doesn't make a difference, but usually the bitwise operators are only used for integers and to combine flags for example for pending intents

Reply:
@philipplackner Updated